installer/roles/mobile-control-panel-setup/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
I wonder should we move this stuff to a fork of the openshift-ansible repo and work on the mobile control panel there. Obviously this is fine now for initial dev